test/metrics.network.test.js

```diff
@@ -0,0 +1,40 @@
+import test from 'ava';
+import Metrics from '../lib/metrics';
+import nock from 'nock';
+
+nock.disableNetConnect();
```

Comment:
should be in a `before`, and removed in an `after`, or your tests have side effects